apps/support/Makefile.inc

```diff
@@ -145,7 +158,7 @@ RUNARGS ?=
 #
 #     make foo.run RUNARGS='input=a output=baz'
 #
-$(BIN)/%.run: $(BIN)/%.rungen
+$(BIN)/$(HL_TARGET)/%.run: $(BIN)/$(HL_TARGET)/%.rungen
```

Comment:
This one is unfortunate - it means I can't say 'make bin/some_target/foo.run' without also setting HL_TARGET to equal the same. Does it work to say `$(BIN)/%.run: $(BIN)/%.rungen`?

Reply:
Even if it did, the rules (above it) for building .rungen and .registration wouldn't work.